execCommand.html

```diff
@@ -639,68 +640,44 @@ <h2 id="methods-to-query-and-execute-commands">Methods to query and
-  readonly attribute DOMString <a href="#dom-editinginputevent-value" title=
-"dom-EditingInputEvent-value">value</a>;
-};
-
-dictionary <dfn id=
-"editinginputeventinit">EditingInputEventInit</dfn> : <a class="external"
-data-anolis-spec="dom" href=
-"http://dom.spec.whatwg.org/#eventinit">EventInit</a> {
-  DOMString command;
-  DOMString value;
-};
-</pre>
+            [Exposed=Window,
+             Constructor(DOMString type, optional EditingBeforeInputEventInit eventInitDict = {})]
+            interface EditingBeforeInputEvent : Event {
+              readonly attribute DOMString command;
+              readonly attribute DOMString value;
+            };
+            
+            dictionary EditingBeforeInputEventInit : EventInit {
+              DOMString command;
+              DOMString value;
+            };
+
+            [Exposed=Window,
+             Constructor(DOMString type, optional EditingInputEventInit eventInitDict = {})]
+            interface EditingInputEvent : Event {
+              readonly attribute DOMString command;
+              readonly attribute DOMString value;
+            };
+            
+            dictionary EditingInputEventInit : EventInit {
+              DOMString command;
+              DOMString value;
+            };
+        </pre>
 
         <p class="XXX">We have two different interfaces because we might want
         to add additional members to the input event but not the beforeinput
         event, such as a list of nodes that were affected.</p>
 
-        <p>When an <code><a href=
-        "#editingbeforeinputevent">EditingBeforeInputEvent</a></code> object is
-        created, the <dfn id="dom-editingbeforeinputevent-command" title=
-        "dom-EditingBeforeInputEvent-command"><code>command</code></dfn> and
-        <dfn id="dom-editingbeforeinputevent-value" title=
-        "dom-EditingBeforeInputEvent-value"><code>value</code></dfn> attributes
-        must both be initialized to the empty string, unless otherwise
-        specified.</p>
-
-        <p>When an <code><a href=
-        "#editinginputevent">EditingInputEvent</a></code> object is created,
-        the <dfn id="dom-editinginputevent-command" title=
-        "dom-EditingInputEvent-command"><code>command</code></dfn> and <dfn id=
-        "dom-editinginputevent-value" title=
-        "dom-EditingInputEvent-value"><code>value</code></dfn> attributes must
-        both be initialized to the empty string, unless otherwise
-        specified.</p>
+        <p data-dfn-for="EditingBeforeInputEvent">When an <a>
+        EditingBeforeInputEvent</a> object is created, the <dfn>command</dfn>
+        and <dfn>value</dfn> attributes must both be initialized to the empty
+        string, unless otherwise specified.</p>
+
+        <p data-dfn-for="EditingInputEvent">When an <a>EditingInputEvent
+        </a> object is created, the <dfn>command</dfn> and <dfn>value</dfn>
+        attributes must both be initialized to the empty string, unless
```

Comment:
Unrelated, but I wonder if the IDL should just EventInputEventInitDict should default the corresponding members to `= ""`. Initializing them in prose is not ideal.

Reply:
Can the prose be removed then, for brevity?

Reply:
Absolutely!